recipes/openmesh/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -std=c++11"
+  export LINKFLAGS="${LDFLAGS}"
+  export MACOSX_DEPLOYMENT_TARGET=10.7
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
+
+make install -j${CPU_COUNT} VERBOSE=1
```

Comment:
Be careful using the `CPU_COUNT` on the CI as it often misreports. I think, in general, we're better of just building single threaded if we can.